src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistActiveToggle.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { Toggle } from '~core/Fields';
+import Heading from '~core/Heading';
+
+import styles from './ManageWhitelistDialogForm.css';
+
+const displayName = `dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.ManageWhitelistActiveToggle`;
+
+const MSG = defineMessages({
+  toggleLabel: {
+    id: `dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.ManageWhitelistActiveToggle.toggleLabel`,
+    defaultMessage: `{whitelistStatusValue, select,
+      true {Active}
+      other {Inactive}
+    }`,
+  },
+  headerTitle: {
+    id: `dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.ManageWhitelistActiveToggle.headerTitle`,
+    defaultMessage: 'Whitelisted addresses',
+  },
+  tooltipText: {
+    id: `dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.ManageWhitelistActiveToggle.tooltipText`,
+    defaultMessage: `Whitelist is active by default once at least one address is added to the list. You can turn this feature “Off” to deactivate the whitelist. Use with caution.`,
+  },
+  warningText: {
+    id: `dashboard.ManageWhitelistDialog.ManageWhitelistDialogForm.ManageWhitelistActiveToggle.warningText`,
+    defaultMessage: `<span>Warning.</span> You have deactivated the whitelist. You acknowledge potential risks and consequences by clicking ‘Confirm’. `,
+  },
+});
+
+interface Props {
+  whitelistStatusValue: boolean;
+}
+
+const ManageWhitelistActiveToggle = ({ whitelistStatusValue }: Props) => (
+  <>
+    <div className={styles.toggleContainer}>
+      <Heading
+        appearance={{ size: 'normal', margin: 'none', theme: 'dark' }}
+        text={MSG.headerTitle}
+      />
+      <Toggle
+        label={MSG.toggleLabel}
+        labelValues={{ whitelistStatusValue }}
+        name="whitelistStatus"
```

Comment:
Same here whitelistStatus with values true or false, it doesn't tell too much. Again maybe something like whitelistActivated (true/false)